project5/3700kvstore

```diff
@@ -237,7 +280,7 @@ class Server(LoggingMixin):
 
     def reset_timeouts(self):
         """Resets the election and leader timeouts"""
-        self.election_timeout = random.uniform(0.15, 0.3)
+        self.election_timeout = random.uniform(0.45, 0.6)
```

Comment:
Is there a reason you increased these times?

Reply:
Yeah the times in the paper (where I think 150 - 300 ms comes from) assume you're on a fairly reliable Ethernet, not an intentionally shitty network

Reply:
Lol okay